google/cloud/security/common/gcp_api/admin_directory.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, global_configs):
             api_name=self.API_NAME)
 
         self.rate_limiter = RateLimiter(
-            self.global_configs.get('max_admin_api_calls_per_day'),
+            self.global_configs.get('max_admin_api_calls_per_100_seconds'),
```

Comment:
I have some ideas that I've sent out in a doc. Probably doesn't need to be implemented in this PR, but it would be nice to nail down a process for the future.